metagraph/tests/graph/succinct/test_boss.cpp

```diff
@@ -1076,7 +1076,8 @@ TEST(BOSS, CallUnitigsTwoBigLoops) {
             num_kmers += path.size();
         }, num_threads, 0, true);
 
-        EXPECT_EQ(2, num_sequences);
+        // There is no guarantee on the number of contigs in the primary mode
+        // EXPECT_EQ(2, num_sequences);
         EXPECT_EQ(sequences[0].size() - k - 1 + sequences[1].size() - k - 1 - 1,
                   num_kmers);
```

Comment:
there's something strange here now. If `num_sequences` is no longer 2, then the next test should either fail as well, or we're outputting duplicate k-mers

Reply:
no, it's ok. `sequences` contains the input sequences, there are just two of them.

Reply:
ah, sorry, my bad. For some reason, I read this as if it were referring to the sequences extracted from the graph.

Reply:
yeah, at first I also thought so, I guess it won't hurt if we rename it